### PR TITLE
update of screenshots for enterprise tenants

### PR DESCRIPTION
**Pull request by darluk**
removed a couple of screenshots, because the text is quite descriptive by itself

### CODE REVIEW

File: content/benutzerhandbuch/enterprise-tenant-de-bundle/license-management.md [hunk 1]
```diff
@@ -8,8 +8,6 @@ aliases:
 
 Der {{< management-tenant-de >}} kann die Lizenzen für die Domains der {{< enterprise-tenant-de >}}s über die Benutzeroberfläche verwalten.
 
-![License management](/images/benutzerhandbuch/enterprise-tenant/et-license-management.png)
```
Comment (darluk) [inline content/benutzerhandbuch/enterprise-tenant-de-bundle/license-management.md:11]: this is the original screenshot 
Screenshot from 2023-01-31 00-11-31

File: content/users-guide/enterprise-tenant-bundle/usage-and-billing.md [hunk 2]
```diff
@@ -159,7 +159,6 @@ The date/time range used here might differ from your server time due to differen
 
 1. Click Export CSV at the right of the top menu bar to export the current view of the statistics table to a CSV file.
 2. In the resulting dialog box you can customize the CSV output by specifying a field separator, decimal separator and charset.
-<br> <img src="/images/users-guide/enterprise-tenant/et-subtenant-statistics-export.png"></img> <br>
```
Comment (darluk) [inline content/users-guide/enterprise-tenant-bundle/usage-and-billing.md:162]: the original screenshot 
Screenshot from 2023-01-31 00-12-25

File: content/users-guide/enterprise-tenant-bundle/usage-and-billing.md [hunk 3]
```diff
@@ -252,8 +251,6 @@ For instance, when you have a microservice with scale policy set to AUTO and the
 
 Note that an audit record is created for every change of the number of instances.
 
-![Audit logs](/images/users-guide/enterprise-tenant/et-audit-logs-microscaling.png)
```
Comment (darluk) [inline content/users-guide/enterprise-tenant-bundle/usage-and-billing.md:255]: the original screenshot 
Screenshot from 2023-01-31 00-13-49

File: content/users-guide/enterprise-tenant-bundle/usage-and-billing.md [hunk 4]
```diff
@@ -365,8 +362,6 @@ Any extension deployed to the platform as a microservice is billed as "used" and
 
 A tenant that is billed for resources can view the point in time when the microservices billing has been changed in [the audit logs](/users-guide/administration/#audit-logs). The audit log entries, for example "Scaling application '...' from X to Y instances" contain the information about the changes of instances and resources consumed by the microservice.
 
-  <img src="/images/users-guide/enterprise-tenant/et-ms-billing-audit-logs.png" name="Microservice audit logs"/>
```
Comment (darluk) [inline content/users-guide/enterprise-tenant-bundle/usage-and-billing.md:368]: screenshot contains bug, which is not fixed yet 
Screenshot from 2023-01-31 00-14-53

**Review by jkot-softwareag — APPROVED** [review]
my only concern is about domains in this screenshot:
https://github.com/SoftwareAG/c8y-docs/pull/818/files#diff-de76c3ede38e1c3c581df7a39b0c4d4134fc31226afe0ea279ff769f4e5474c1

I am not sure if we can do something about `test-tenant1.latest.stage.c8y.io`. Maybe try find some EE tenant in this env which will not have such "test" domain additionally exposing stage info?

**Comment by darluk** [comment]
> test-tenant1.latest.stage.c8y.io

I've added test-tenant1/ test-tenant2 because didn't know how to name tenants, but it could be easily changed, Do you have some proposition on how it should be named? but there is no way to exclude `latest.stage.c8y.io` only If will be some domain change, but not sure how to do it, hence not sure even if should I spend more time on the ticket

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
> > test-tenant1.latest.stage.c8y.io
> 
> I've added test-tenant1/ test-tenant2 because didn't know how to name tenants, but it could be easily changed, Do you have some proposition on how it should be named? but there is no way to exclude `latest.stage.c8y.io` only If will be some domain change, but not sure how to do it, hence not sure even if should I spend more time on the ticket

@darluk @jkot-softwareag Couldn´t you fake this in the source file for the purpose of the screenshot? I´ve seen Enio doing that :-). You are right, Jakub, it is not perfect, but I would rate this as minor and leave as is, if there is no quick solution.

**Comment by jkot-softwareag** [comment]
> @darluk @jkot-softwareag Couldn´t you fake this in the source file for the purpose of the screenshot? I´ve seen Enio doing that :-). You are right, Jakub, it is not perfect, but I would rate this as minor and leave as is, if there is no quick solution.

We had a small chat about it, maybe Daria will find few tenants with different domains on one of stagings. I found 2 with eg. domain "acme.com" which for me looks better than "test.staging.c8y.io". I agree that this is minor and I approved this PR.

**Review by karolkaczmarski-sag — APPROVED** [review]
screenshots should have 1440px width

**Comment by darluk** [comment]
> screenshots should have 1440px width

isn't it a max size? but ok, will update to have 1440 for all

**Comment by BeateRixen** [comment]
> > screenshots should have 1440px width
> 
> isn't it a max size? but ok, will update to have 1440 for all

As just discussed, 1440 px is the correct width but if it is slightly smaller we can live with it :-)

**Pull request merged**